README.md

```diff
@@ -20,7 +20,8 @@ The action also accepts some optional input parameters:
 * `dockerRunArgs`: Additional arguments to pass to `docker run`, such as volume mappings. See [`docker run` documentation](https://docs.docker.com/engine/reference/commandline/run).
 * `setup`: Shell commands to execute on the host before running the container, such as creating directories for volume mappings.
 * `install`: Shell commands to execute in the container as part of `docker build`, such as installing dependencies. This speeds up subsequent builds if `githubToken` is also used, but note that the image layer will be publicly available in your projects GitHub Package Registry, so make sure the resulting image does not have any secrets cached in logs or state.
-* `base_image`: Specify a custom base image, such as [busybox](https://hub.docker.com/_/busybox), `arch` and `distro` should be set to `none` in this case. This will allow you to chose direcly the image that will be used in the *FROM* clause of the internal docker container without needing to create a Dockerfile.arch.distro for a specific arch/distro pair. For more detials, see [PR #103](https://github.com/uraimo/run-on-arch-action/pull/103#issuecomment-1363810049). Known limitation: Only one base_image configuration for each workflow if you use GitHub images caching.
+* `base_image`: Specify a custom base image, such as [busybox](https://hub.docker.com/_/busybox), `arch` and `distro` should be set to `none` in this case. This will allow you to chose direcly the image that will be used in the *FROM* clause of the internal docker container without needing to create a Dockerfile.arch.distro for a specific arch/distro pair. For more details, see [PR #103](https://github.com/uraimo/run-on-arch-action/pull/103#issuecomment-1363810049). Known limitation: Only one base_image configuration for each workflow if you use GitHub images caching.
+* `qemu_static_image`: Specify a custom image for registering QEMU emulators. If not set, **multiarch/qemu-user-static** will be used by default.
```

Comment:
Would that work with aptman/qus (https://dbhi.github.io/qus/) ?
AFAIK QUS is also still on QEMU 7.1 but I wonder how generic this new setting is - does it support any qemu-user-static provider or just forks of multiarch/qemu-user-static.

One thing I like better in QUS is that it works on other platforms too, e.g. you can emulate x86_64 on aarch64 machine.

Reply:
For now, I have tested with [my own fork of multiarch/qemu-user-static](https://github.com/LonghronShen/qemu-user-static), which can be used from [longhronshens/qemu-user-static:latest](https://hub.docker.com/r/longhronshens/qemu-user-static/tags). This fork has been updated to use qemu 7.2. As for **aptman/qus**, this is more complicated than what **run-on-arch-action** does. That is not very good for simple cases. This is why I'd rather to use **run-on-arch-action**.

Reply:
As for **emulating x86_64 on aarch64 machine**, I think my fork can also be used like that, since I have pushed [qemu-user-static:latest with multiple architectures](https://hub.docker.com/r/longhronshens/qemu-user-static/tags). But since **uraimo/run-on-arch-action** can only be used on GitHub Action runner which runs on x86_64 by default, multi-arch support is somewhat meaningless.